tests/test_lab_config.py

```diff
@@ -22,6 +75,33 @@ def test_ilab_config_show(cli_runner: CliRunner) -> None:
     assert configuration.Config(**parsed)
 
 
+def test_ilab_config_show_key_general_yaml(
+    cli_runner: CliRunner, tmp_path: pathlib.Path
+) -> None:
+    test_config_path = tmp_path / "test_config.yaml"
+    test_config_path.write_text(test_source_config_data)
+
+    cfg = configuration.read_config(str(test_config_path))
```

Comment:
So far, I think is the same. Your previous reply reminds me: Or whatever is the smallest amount of config that isn't likely to change.
If depends on the default config  that may change in the future, maybe just maybe the test will not work. So created a source data and make it stable.
Please correct me if I am wrong.

Reply:
One of my previous comments was about hardcoding a section of the config in a str and then comparing that to the default.  I was worried about that being finicky because you would have to change it each time the config changed.  I think if you do something similar to what I suggest above, it will always work as long as the `general` key is there and the formatting adjustments you have are generic enough.

Note that with the code the way you have it now, it's not really isolating itself from the current config because new values will still come in from config.  I think it only has an effect for what it overrides.